src/ansible_creator/resources/common/role/roles/run/README.md.j2

```diff
@@ -1,4 +1,4 @@
-{{ namespace|capitalize }}.{{ collection_name|capitalize }} Run Role
+{{ namespace|capitalize }}.{{ collection_name|capitalize }} {{ role_name }} Role
```

Comment:
Let's not capitalize here and instead:

Role: {{ namespace }}.{{ collection_name }}.{{ role_name }}